### ISSUE TITLE
when use multigpu, the training is very slow

### ISSUE THREAD

**Issue by kasyoukin** [ZJULearning/pixel_link#52]
./scripts/train.sh 0,1,2 8
step 40: loss = 1.9969 (4.166 sec/step)

```
|   0  GeForce GTX TIT...  Off  | 00000000:04:00.0 Off |                  N/A |
| 34%   71C    P2    89W / 250W |  10516MiB / 12212MiB |      9%      Default |
+-------------------------------+----------------------+----------------------+
|   1  GeForce GTX TIT...  Off  | 00000000:08:00.0 Off |                  N/A |
| 29%   68C    P2    82W / 250W |  10516MiB / 12212MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   2  GeForce GTX TIT...  Off  | 00000000:84:00.0 Off |                  N/A |
| 31%   71C    P2    93W / 250W |  10516MiB / 12212MiB |     28%      Default 
```

all gpu is not busy, i try to modify the num_readers and num_preprocessing_threads ,but it is not work

**Comment by ICTwangbiao** [comment]
@kasyoukin Pity, I have the same problem too, have you solved this problem?
